core/classifiers/CodeClassifier/CodeClassifier.py

```diff
@@ -538,6 +538,20 @@ def __init__(self):
         # text into a feature a vector.
         self.count_vector = None
 
+    @property
+    def name_in_job_result_proto(self):
+        # This is just a temporary fix to get lint tests passing.
+        # CodeClassifier will be re-invented before enabling it, at some point
```

Comment:
Be more explicit here about: (a) what lint test is failing otherwise, (b) what specific action needs to be done to get this to work properly in the "correct"/"ideal" case.

Ditto below.

Reply:
Modified comment. Now it should explain the situation better.